rclcpp/include/rclcpp/node.hpp

```diff
@@ -153,6 +155,9 @@ class Node : public std::enable_shared_from_this<Node>
   const std::vector<rclcpp::CallbackGroup::WeakPtr> &
   get_callback_groups() const;
 
+  RCLCPP_PUBLIC
+  void for_each_callback_group(const CallbackGroupFunction & func);
```

Comment:
We may as well make this one as much like the one on the `master` branch as possible.  That means copying over the style and the docblock as well (the only thing that should be different in the end is the exact namespace that `CallbackGroupFunction` is in).